### PR TITLE
fix(linux): skip oculus library detection

### PR DESCRIPTION
**Pull request by RedlineTriad**
## Scope
This fixes a startup exception on linux, due to attempted windows registry read.
The combination of this and #264 makes downloading songs error free on linux.

## Implementation
Currently I just skip Oculus library detection since there are very few oculus users on linux.
However, just in case somebody wants to know why it isn't working I left a log message.

I can also add a `// TODO` or we could also open a ticket, but I am lazy.

## How to Test
Open client on linux and observe lack of exception compared to master branch.

## Emoji Guide

**For reviewers: Emojis can be added to comments to call out blocking versus non-blocking feedback.**

E.g: Praise, minor suggestions, or clarifying questions that don’t block merging the PR.

> 🟢 Nice refactor!

> 🟡 Why was the default value removed?

E.g: Blocking feedback must be addressed before merging.

> 🔴 This change will break something important

| | | |
| --- | --- | --- |
| Blocking | 🔴 ❌ 🚨 | RED |
| Non-blocking | 🟡 💡 🤔 💭 | Yellow, thinking, etc |
| Praise | 🟢 💚 😍 👍 🙌 | Green, hearts, positive emojis, etc |

### CODE REVIEW

**Review by Zagrios — APPROVED** [review]

**Comment by RedlineTriad** [comment]
Is this waiting on anything before it can get merged?

**Comment by RedlineTriad** [comment]
Updated to fix merge conflicts after prettify.

**Comment by Zagrios** [comment]
thanks 🙌

**Pull request merged**
